.evergreen/functions.yml

```diff
@@ -690,6 +690,8 @@ functions:
           MCLI_ORG_ID: ${e2e_tests_mcli_org_id}
           MCLI_PROJECT_ID: ${e2e_tests_mcli_project_id}
           MCLI_OPS_MANAGER_URL: ${e2e_tests_mcli_ops_manager_url}
+          # TODO: remove when issues with docker are resolved
+          DOCKER_REGISTRY: artifactory.corp.mongodb.com/dockerhub
```

Comment:
I know the plan is to just skip those in CI while we're waiting for docker user creds, but these tests are especially valuable while we're approaching the feature release, so I'd prefer for them to continue functioning

(I haven't found a ticket to link in the TODO yet)